e2e/js_run_devserver/src/BUILD.bazel

```diff
@@ -47,6 +57,7 @@ js_run_devserver(
         ":http_root",
         "//:node_modules/http-server",
     ],
+    log_level = "debug",
```

Comment:
Intentional?

Reply:
Yes, if these tests ever fail then its nice to have debug logs in the output already on